### ISSUE TITLE
About the loss

### ISSUE THREAD

**Issue by door5719** [springkim/YOLOv3_SpringEdition#3]

**Comment by door5719** [comment]
I'm very glad to get your reply.If you finish implementing that paper,please give me some advice.Thank you very much.

**Comment by door5719** [comment]
Hi@springkim, in your work about yolov3,yolo_layer.c,from line 167 to 174:
for(t = 0; t < l.max_boxes; ++t){
                        box truth = float_to_box(net.truth + t*(4 + 1) + b*l.truths, 1);
                        if(!truth.x) break;
                        float iou = box_iou(pred, truth);
                        if (iou > best_iou) {
                            best_iou = iou;
                            best_t = t;
                        }
I don't very clear what  the  "  box truth"represents for.

**Comment by springkim** [comment]
Hi @door5719 ,
Truth means Ground Truth(GT). We have to compare with GT to get the loss of the prediction box, but there are many prediction boxes and GT's on the image. So we use IOU to see which predictive boxes and GT are responsive.

**Comment by door5719** [comment]
@Thank you very much

**Comment by door5719** [comment]
@springkim  thank you very much

**Comment by door5719** [comment]
@springkim ,I want change the loss function partly. I load a yolov3 model as a detecting net(the detected result will  be used for traing current net ,current net be trained is yolov3tiny ) .   If b.box part of original Loss function can be wrote down as a function: f(b_gt,b_pr),b_gt means box of Ground Truth ,b_pr means prediction box  of current net
result, how can i change it  as a function: 0.3*(b_gt,b_pr)+0.7*f(b_t,b_pr)? b_t means result   detected from yolov3 ? Thank you for your reading my question.
